src/components/Sidebar/Sidebar.scss

```diff
@@ -1,2 +1,29 @@
 //@import '../../styles/variables';
 //@import '../../styles/mixins';
+
+.sidebar {
+  position: fixed;
+  width: 280px;
+  max-width: 70%;
+  height: 100%;
+  left: 0;
+  top: 0;
+  padding: 20px;
+  background-color: white;
+  z-index: 200; 
```

Comment:
move all z-indeces to variables to one place and name it properly. It will be useful when adding and handling more indeces to the app later on.